xss/xss.go

```diff
@@ -128,6 +128,23 @@ func bufferedBodyHandler(w http.ResponseWriter, r *http.Request, safety string)
 	return template.HTML(input), false
 }
 
+func responseHandler(w http.ResponseWriter, r *http.Request, safety string) (template.HTML, bool) {
+	userInput := utils.GetUserInput(r)
+	switch safety {
+	case "safe":
+		userInput = url.QueryEscape(userInput)
+		w.Write([]byte(userInput))
+	case "unsafe":
+		w.Write([]byte(userInput))
```

Comment:
For this ticket I was hoping to see net/http policy, Response.Write(), trigger; but, this work will trigger ResponseWriter.Write().

Would it be possible to update so we use the Response receiver specifically, instead of ResponseWriter?
https://golang.org/pkg/net/http/#Response.Write